src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.netcoreapp.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml;
+using System.Xml.Serialization;
+using Xunit;
+
+public static partial class XmlSerializerTests
+{
+
+#if ReflectionOnly
```

Comment:
since this is the only thing in the file now, can the #if be changed to a condition in the project maybe?